lib/modules/manager/fleet/extract.ts

```diff
@@ -118,7 +118,7 @@ function extractFleetFile(doc: FleetFile): PackageDependency[] {
       result.push({
         // overwrite name with customization name to allow splitting of PRs
         ...dep,
-        depName: custom.name,
+        depName: custom.name ?? `targetCustomization-${index + 1}`, // if no name is provided, use the index
```

Comment:
```suggestion
        depName: custom.name ?? `targetCustomization-${index}`, // if no name is provided, use the index
```
i see no need for this increment.

Reply:
@rarkins why did you suggest the index increment? 🤔

Reply:
its just easier for humans. Maybe `targetCustomization[${index}]` to show that it's an array and should be counted from 0?